blog/2023-05-31-may-releases.md

```diff
@@ -0,0 +1,150 @@
+* Retry language in flexible solve and `repodata` logs are now less ominous.
+* Python 3.11 is now supported.
+
+
+### 🔧 What Got Fixed? 🔧
+
+The following bug fixes were implemented in the 23.5.0 version of conda:
+
+* `conda clean` no longer fails if it was unable to get the file stats.
+* If `conda.deprecations.DeprecationHandler` receives a bad version, a fallback version is provided.
+* The default value for `defaults` includes `msys2` when `context.subdir` is `win-*` on non-Windows platforms.
+* `TypeError`s are avoided when non-string types are written to the index cache metadata.
+* `conda.core.package_cache_data.UrlsData.get_url` no longer fails when `package_path` has a `.conda` extension.
+* No more pre-converting of paths to Unix style on Windows in `conda.sh`; this is so that they are prefix-replaceable upon installation.
+
+
+### 🌅 What's Marked for [Deprecation](https://github.com/conda-incubator/ceps/blob/main/cep-9.md)? 🌅
+
+The following modules and functions were marked for deprecation:
+
+* [`conda_env.pip_util.get_pip_version`](https://github.com/conda/conda/pull/12492)
+* [`conda_env.pip_util.PipPackage`](https://github.com/conda/conda/pull/12492)
+* [`conda_env.pip_util.installed`](https://github.com/conda/conda/pull/12492)
+* [`conda_env.pip_util._canonicalize_name`](https://github.com/conda/conda/pull/12492)
+* [`conda_env.pip_util.add_pip_installed`](https://github.com/conda/conda/pull/12492)
+* [`conda_env.env.load_from_directory`](https://github.com/conda/conda/pull/12492)
+* [`python -m conda_env.cli.main`](https://github.com/conda/conda/pull/12492) (use `conda env` instead)
+* [`python -m conda_env`](https://github.com/conda/conda/pull/12492) (use `conda env` instead)
+* [`conda.auxlib.packaging`](https://github.com/conda/conda/pull/12509)
+* [`conda.testing.integration.get_conda_list_tuple`](https://github.com/conda/conda/pull/12676) (use `conda.core.prefix_data.PrefixData().get()` instead)
+* [`conda.testing.encode_for_env_var`](https://github.com/conda/conda/pull/12677)
+* [`conda.testing.integration.temp_chdir`](https://github.com/conda/conda/pull/12678) (use `monkeypatch.chdir` instead)
+
+* * *
+
+## [Changes in Conda-Build](https://github.com/conda/conda-build/releases/tag/3.25.0)
+
+To update conda-build, run:
+```bash
+conda install -n base conda-build=3.25.0
+```
+
+### ✨ What's New? ✨
+
+In the latest version of conda-build, noarch packages that use virtual packages can now have them added to the hash contents of the package. This facilitates the building of noarch packages multiple times for different platforms with platform-specific dependencies. In conda-build 3.25.0, different variants can be built for `__linux`/`__osx`/`__win` and get non-clashing package file names.
+
+Additionally, the following features and changes can be found in conda-build 3.25.0:
+
+* Support for `svn` source credentials (`svn_username` and `svn_password`).
```

Comment:
Link to the updated docs: https://docs.conda.io/projects/conda-build/en/3.25.x/resources/define-metadata.html#source-from-svn